Azure Template CI.yaml

```diff
@@ -328,10 +338,3 @@ jobs:
       ArtifactName: Solution Template
       FileCopyOptions: ''
 
-  - task: AzureResourceManagerTemplateDeployment@3
-    condition: succeeded()
-    inputs:
-      deploymentScope: 'Resource Group'
-      ConnectedServiceName: $(ConnectedServiceName)
-      action: 'DeleteRG'
-      resourceGroupName: BakingDP-TemplateTest-$(Build.BuildNumber)
```

Comment:
I said to duplicate the task, not move it. You need to re-instate this task as it is.

Reply:
You have still not done this!